### PR TITLE
Adding ability to log meeting attendance and current movie

### PR DESCRIPTION
**Pull request by BrandonCKrueger**
Just getting this guy started -- what are your thoughts @charwking?

Basically when a meeting is hosted, the `presentUsers`, `currentMovie.name`, and `currentMovieUser.userId` are all saved in the database at `firebase/meetings/{meeting date}`.

This should give us the ability to comb attendance records to do the weighting.

I am stuck (since I don't know how to use firebase).  When I run the code, it says 'access denied' for creating the `firebase/meetings/{date}` child node.

### CODE REVIEW

**Comment by charwking** [comment]
Summarizing the [README](https://github.com/charwking/movie-club#firebase), you have a couple options:
* Update the `FIREBASE.md` doc and I'll apply those rules to the dev environment and you can continue from there.
* Create your own firebase instance using the rules in `FIREBASE.md` as a baseline and use it for testing.
* Wait until I've got enough free time to determine what the rules should be and I'll update them in dev.

Let me know if you have questions.

**Comment by BrandonCKrueger** [comment]
Being the first time I've done anything with firebase so the rules might need a looking over before applying them but I think I was able to reverse engineer the other rules.

**Comment by charwking** [comment]
I made some changes to the firebase rules and documented them in PR #135. Let me know if you have any questions.

**Comment by charwking** [comment]
Sorry @BrandonCKrueger -- I forgot to mention that the Firebase rules I added will necessitate some code changes. If you want some help -- or want to suggest some different rules -- let me know!

**Comment by BrandonCKrueger** [comment]
I'm not actually sure if it's saving to firebase, but the logic is pushing data that is inline with the new firebase rules:
```
var data = {
    presentUsers: presentUsers,
    selectedMovieName: selectedMovieName,
    selectedMovieUserId: selectedMovieUserId
};
firebaseRef.child('meetings').child(formatDate(date)).set(data);
```

Example of data:
```
{
    presentUsers: {'simpleLogin:24': true, 'simpleLogin:51': true },
    selectedMovieName: 'Awesome video of Brandon',
    selectedMovieUserId: 'simpleLogin:24'
}

**Comment by charwking** [comment]
Looks like you've got some merge conflicts, but a quick look didn't reveal anything that caused me concern. I added you as a contributor to the repo, so no more waiting on me to merge your stuff. Welcome aboard sir, keep up the good work!

**Comment by BrandonCKrueger** [comment]
It was that darn `firebase.md` file!  I finally got it to realize master had stuff to merge in.

I'll finish this guy up sometime over the weekend.

File: src/hostMeeting/hostMeeting.controller.js [hunk 1]
```diff
@@ -81,6 +85,69 @@
...
+        function selectWinner() {
+            var deferred = $q.defer();
+            meetingApi.getAll().$loaded().then(function (meetings) {
+                var chances = {};
+                getPossibleWinners(chances);
+                generateProbabilitiesFromMeetings(chances, meetings);
+                var winner = selectRandomWinner(chances);
+                deferred.resolve(winner);
+            });
+            return deferred.promise;
```
Comment (charwking) [inline src/hostMeeting/hostMeeting.controller.js]: Using `$q.defer()` is rarely needed, and is a very common promise anti-pattern. Try this as an alternative:

```javascript
function selectWinner() {
   return meetingApi.getAll().$loaded().then(function (meetings) {
      var chances = {};
      getPossibleWinners(chances);
      generateProbabilitiesFromMeetings(chances, meetings);
      return selectRandomWinner(chances);
   });
}
```

Comment (BrandonCKrueger) [inline src/hostMeeting/hostMeeting.controller.js]: Wooooooaaaaaahhhhhh

**Pull request merged**